index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Employment and Entrepreneurship Fair 2022</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+  <link rel="stylesheet" href="https://use.typekit.net/cxn7bfv.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">             
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="nav-bar-mobile">
+        <i class="fa-sharp fa-solid fa-bars fa-2x" id="hamburger"></i>
+      </div>
+      <div class="nav-bar-desktop">
+        <a href="#presentation"><img class="logo-img" src="https://drive.google.com/uc?export=download&id=1kJyz0MXkWANEBI2fDUfR_w0omliq9Plp" alt="Employment and Entrepreneurship Fair 2022 logo"></a>
+        <ul class="nav-bar-desktop-ul">
+          <li class="nav-bar-desktop-list"><a href="/about.html">About</a></li>
+          <li class="nav-bar-desktop-list"><a href="#program">Program</a></li>
+          <li class="nav-bar-desktop-list"><a href="#speakers">Speakers</a></li>
+          <li class="nav-bar-desktop-list"><a href="#sponsors">Sponsors</a></li>
+          <li class="nav-bar-desktop-list1"><a href="#contact">Contact</a></li>
+        </ul
```

Comment:
- [ ] The navbar on the desktop is divided into a top dark bar and a bottom white bar. The top short bar section has social media links, logout, and language links aligned to the right. Please kindly add it 🙃 🙃 

Your design 

<img width="986" alt="Screen Shot 2022-10-20 at 23 30 26" src="https://user-images.githubusercontent.com/74501523/197071848-d024c918-f965-42f5-ba2a-c07ce783fd10.png">

Original design 

<img width="986" alt="Screen Shot 2022-10-20 at 23 29 48" src="https://user-images.githubusercontent.com/74501523/197072011-be0b523a-db0d-4b85-94a0-eabbb5e59d7d.png">